crawler/db.py

```diff
@@ -259,6 +261,72 @@ def run_mysql_executemany_query(mysql_conn: CMySQLConnection, sql_query: str, va
         mysql_conn.close()
 
 
+def run_mysql_execute_formatted_query(
```

Comment:
Would you be able to add some tests for this, if you haven't already?

Reply:
I've added two tests, similar to those for the executemany query